### DISCUSSION TITLE
Registry 3.0 ENV variables

### DISCUSSION THREAD

**Discussion by ottuzzi-at-vegans** [Apicurio/apicurio-registry#5192] [Q&A] [answered]
I'm really sorry for the dumb question but I did not find a solution for running V3 on remote server.
I started both registry and registry-ui (M4 release) using docker but when I connect from my machine to remote registry-ui (http://18.199.224.164:8888/explore) I get the error **Connection to the Registry server failed** as it tries to find registry API on localhost:8080. Is there an updated env variable list (I saw the 2.6 list in documentation but I think that UI and API were not splitted) in which I can find how to point the UI to the right registry API URL (http://18.199.224.164:8080)?

Thanks in advance
Piero

**Comment by EricWittmann** [comment, marked as answer]
Hi @ottuzzi-at-vegans - no worries, we need to update our documentation for 3.0 to make everything easier.  We have a table of ENV vars for 3.0, but not yet published.  For now, let me just explain. :)

The UI container image needs to be configured with the location of the API component.  For all other config options, the UI will load the values from the API component.  As a result, the **only** config option you should use for the UI container is:

`REGISTRY_API_URL`

Where the value should be the URL of the API component's REST API.  For example:

`https://registry-api.example.org/apis/registry/v3`
